pom.xml

```diff
@@ -33,19 +33,18 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
-<!--		<dependency>-->
-<!--			<groupId>org.apache.derby</groupId>-->
-<!--			<artifactId>derby</artifactId>-->
-<!--			<scope>runtime</scope>-->
-<!--		</dependency>-->
```

Comment:
do we need this commented code?

Reply:
I honestly thought I deleted it. Anyway, at every addition of functionality, I set out to clean the rest of the code or to make a branch just for that.

Reply:
That's why I deleted it. The code is in red.